build/linux/dist/appdata.xml

```diff
@@ -1,8 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- See https://wiki.gnome.org/GnomeGoals/AppDataGnomeSoftware -->
-<application>
-  <id type="desktop">arduino.desktop</id>
-  <licence>CC-BY-SA</licence>
+<!-- See https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html -->
+<component type="desktop-application">
+  <id>cc.arduino.arduinoide.desktop</id>
+  <metadata_license>FSFAP</metadata_license>
```

Comment:
Yes, it is well-defined and recommended for these texts, as it does not require a large license text to be shipped with the metainfo file and because it is very short. See https://spdx.org/licenses/FSFAP.html

In any case, I did not intend to change the license, that was by accident - CC-BY-SA-3.0 is totally fine and I changed it back in the new revision.